docs/notebooks/Testing GP model.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b395dedd-ae15-4788-9911-b9050e3ff784",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/michael/.pyenv/versions/3.9.8/envs/ohbm_venv/lib/python3.9/site-packages/pandas/compat/__init__.py:124: UserWarning: Could not import the lzma module. Your installed Python is incomplete. Attempting to use lzma compression will result in a RuntimeError.\n",
+      "  warnings.warn(msg)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "import shutil\n",
+    "import warnings\n",
+    "\n",
+    "from dipy.core.gradients import gradient_table\n",
+    "from sklearn.gaussian_process import GaussianProcessRegressor\n",
+    "\n",
+    "from eddymotion import dmri\n",
+    "from eddymotion.viz import plot_dwi\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "750f9a01-aac3-452e-8fec-0fcc4034d448",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "base_dir = Path(\"/Users/michael/projects/datasets/ds000206\")\n",
+    "bids_dir = base_dir / \"bids\"\n",
+    "derivatives_dir = base_dir / \"dmriprep\"\n",
+    "\n",
+    "dwi_file = bids_dir / \"sub-05\" / \"ses-JHU1\" / \"dwi\" / \"sub-05_ses-JHU1_acq-GD72_dwi.nii.gz\"\n",
```

Comment:
Does this file contain *b=0* volumes? I believe it should not (i.e., our dwi object doesn't filter them out)

Reply:
Yes it does. I think the load function filters out b values less than 50. 

https://github.com/nipreps/eddymotion/blob/main/eddymotion/dmri.py#L224-L226

Reply:
We just realized that the load function as it currently stands affects the AverageB0 model since no b0s will be found. Will address in a separate PR.

Reply:
Is there an AverageB0 model? I think it's AverageDWI

Reply:
Meaning, it actually doesn't expect bzeros